lib/encoder_impl.h

```diff
@@ -60,6 +60,7 @@ class encoder_impl : public encoder
 	int extent;
 	int event;
 	int location;
+	int count;
```

Comment:
It might make sense to rename this to `d_g3_counter` to match the variable names for the other group types defined just below.